libs/vkd3d/memory.c

```diff
@@ -743,6 +743,9 @@ HRESULT vkd3d_memory_allocator_init(struct vkd3d_memory_allocator *allocator, st
     }
 
     vkd3d_va_map_init(&allocator->va_map);
+
+    allocator->vkd3d_queue = d3d12_device_allocate_vkd3d_queue(device,
```

Comment:
Is this queue ever released?

Reply:
No, because the memory allocator gets destroyed after the queues.

Reply:
I suppose allocate_vkd3d_queue just bumps the virtual queue count, and doesn't really *own* anything anyways?